### PR TITLE
Parsing default values from routes

### PR DESCRIPTION
**Pull request by qkdreyer**

### CODE REVIEW

File: Extractor/Handler/PhpDocHandler.php [hunk 1]
```diff
@@ -107,6 +107,14 @@ public function handle(ApiDoc $annotation, array $annotations, Route $route, \Re
             }
         }
 
+        foreach ($route->getDefaults() as $var => $default) {
+            if (strpos($var, '_') !== 0) {
```
Comment (BigZ) [inline Extractor/Handler/PhpDocHandler.php]: yoda conditions are adivsed

File: Extractor/Handler/PhpDocHandler.php [hunk 2]
```diff
@@ -107,6 +107,14 @@ public function handle(ApiDoc $annotation, array $annotations, Route $route, \Re
...
+                if (!isset($annotationRequirements[$var]['default'])) {
```
Comment (BigZ) [inline Extractor/Handler/PhpDocHandler.php]: you could use empty instead of !isset
that wouls be more easy to read

**Comment by GuilhemN** [comment]
2.x is no longer maintained, please consider upgrading to 3.x.

**Comment by BigZ** [comment]
thanks for updating. i checked out 3.x, good job :)

**Pull request closed without merging**